vxmessenger/transport.py

```diff
@@ -19,12 +21,13 @@ class MessengerTransportConfig(HttpRpcTransport.CONFIG_CLASS):
     access_token = ConfigText(
         "The access_token for the Messenger API",
         required=True)
-    app_id = ConfigText(
-        "The app id for the Messenger API",
-        required=False)
```

Comment:
The `app_id` should still exist for the fallback to work. It just shouldn't be required and generally what we'd do here is add something about the field being deprecated in the description.